README.md

```diff
@@ -6,6 +6,44 @@ We are building an ecosystem of preprepocessing pipeline tools for Data Scientis
 and Data Engineers so they may quickly work through the challenge of extracting
 structured data from unstructured raw documents.
 
+## Getting Started
+
+Unstructured's open-source packages currently target Python 3.8. If you are using or contributing
+to Unstructured code, we encourage you to work with Python 3.8 in a virtual environment. You can
+use the following instructions to get up and running with a Python 3.8 virtual environemtn
+with `pyenv-virtualenv`:
+
+#### Mac / Homebrew
+
+1. Install `pyenv-virtualenv` with `brew install pyenv-virtualenv`
+2. Next, use the following commands to add the `pyenv-virtualenv` startup code to `~/.bash_profile`
```

Comment:
These days Mac's try to get you to use zsh rather than bash, and zsh doesn't seem to consume `.bash_profile` by default (I just checked). So for some users this sequence won't have any effect.

Reply:
Should we switch that to `~/.bashrc`? I have `.bash_profile` on mine but I think I turned off `zsh`

Reply:
Switched over to `.bashrc`

Reply:
I'm pretty sure zsh uses `.zshrc`. It'll depend on which shell they are using. `pyenv`'s installation guide gives separate instructions for bash, zsh, and Fish... maybe we should just link to [that](https://github.com/pyenv/pyenv#user-content-set-up-your-shell-environment-for-pyenv)?

Reply:
Switch that over to the link. So many rc files!